Vegeting/Screens/GroupList/Component/ClubListCollectionView.swift

```diff
@@ -59,7 +59,7 @@ class ClubListCollectionView: UICollectionView {
 
 extension ClubListCollectionView: UICollectionViewDelegate {
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        let detailViewController = PostDetailViewController()
+        let detailViewController = PostDetailViewController(club: clubList[indexPath.item])
```

Comment:
p5; 
해당 viewController가 무엇인지 분명하게 나타내주는건 어떨까요?~~ 
```suggestion
        let postDetailViewController = PostDetailViewController(club: clubList[indexPath.item])
```

Reply:
[f5c6363](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/159/commits/f5c6363df18e6a2f30e841c6d6c7da8a7d40cc69)
동의합니다~ 그래도 비슷하게 적는다고 앞에 detail붙이긴 했는데 .. 역시 정확하게 적는게 좋죠! 적용완료